docs/reference/transform/overview.asciidoc

```diff
@@ -7,22 +7,20 @@
 
 beta[]
 
-A _{dataframe}_ is a two-dimensional tabular data structure. In the context of
-the {stack}, it is a transformation of data that is indexed in {es}. For
-example, you can use {dataframes} to _pivot_ your data into a new entity-centric
-index. By transforming and summarizing your data, it becomes possible to
-visualize and analyze it in alternative and interesting ways.
+{transforms-cap} turn your source data into a two-dimensional tabular data 
```

Comment:
> Transforms turn your source data into a two-dimensional tabular data structure. 

Should this be removed as well?  It already exists in https://www.elastic.co/guide/en/elastic-stack-overview/master/ml-dfanalytics.html, which I think is a better home for it.

Reply:
Right! I removed it in my latest commit (6d72bc1).